python/test.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python
+import csv
+from run import Person
+
+
+def make_bool(value):
+    if value == 'Yes':
+        return True
+    if value == 'No':
+        return False
+    return None
+
+
+if __name__ == "__main__":
+    # open up the csv file containing the tests
+    with open('../shared/example_beards.csv') as csv_file:
```

Comment:
![Screenshot from 2019-10-10 09-24-53](https://user-images.githubusercontent.com/13788498/66517642-f78a1700-eb3f-11e9-9d19-eed520e7497b.png)
Can't seem to find the file. Path looks correct tho, unless I'm missing something.

Reply:
Is it sensitive to where it's being run from?

Reply:
Seems to be that we need to be in the python folder before the the `python test.py` runs.

Reply:
yeah, the import at the top of the file is a relative import